src/main/webapp/resources/js/pages/projects/linelist/components/Table/renderers/IconCellRenderer.jsx

```diff
@@ -16,12 +16,12 @@ export class IconCellRenderer extends React.Component {
     data: PropTypes.object.isRequired
```

Comment:
File comments

Reply:
Updated in 987e5853c183d519a811cfe2fd0e381ee3aae5d0